content/MANUAL.md

```diff
@@ -1,61 +1,316 @@
+        <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+        
+        <!-- Demo configuration -->
+        <GitHubDeployment>true</GitHubDeployment>
+    </PropertyGroup>
+</Project>
+```
+
+### Directory.Packages.props
+
+Contains the list of NuGet packages used in your project. Use this to manage package versions centrally:
+
+```xml
+<Project>
+    <ItemGroup>
+        <PackageVersion Include="Fable.Core" Version="4.0.0" />
+        <!-- Add other packages and their versions here -->
+    </ItemGroup>
+</Project>
+```
+
+Adding a package is easy with the `dotnet add package` command:
 
-- `Directory.UserConfig.props` is the file where you are expected to put configuration unique to your project.
+```bash
+# For the main project
+dotnet add src package Fable.Core
 
-- `Directory.Packages.props` contains the list of NuGet packages that are used in this project. You can add or remove packages from this file.
+# For the demo project
+dotnet add demo package Fable.Core
+```
 
-    The easiest way to add a package is to use the `dotnet add package` command. It will automatically add the package to all files that need it.
+### Directory.Build.props
 
-    ```bash
-    # For the main project
-    dotnet add src package Fable.Core
-    # For the demo project
-    dotnet add demo package Fable.Core
-    ```
+Contains default build rules that should work for any project. **Only modify this file if you know what you're doing.**
 
-- `Directory.Build.props` contains a set of default rules that should works for any project. Only modify it if you know what you are doing. ⚠️
+## Development Workflow
 
-The template pre-configures a lot of things for you via the file `Directory.Build.props`, you should not need to modify it. If you do so, please make sure you know what you are doing.
+### Setting Up Your Environment
 
-## Commit convention
+1. Ensure you have the .NET SDK installed
+2. Restore tools: `dotnet tool restore`
+3. Install JavaScript dependencies:
+   ```bash
+   cd demo
+   npm install  # or pnpm install / yarn
+   ```
 
-This repository is set to generate release and changelog based on the commit history.
+### Creating Your Binding
 
-To make sure, you follow the convention, it is configured to use [EasyBuild.CommitLinter](https://github.com/easybuild-org/EasyBuild.CommitLinter) to validate the commit messages.
+1. Edit the files in the `src/` directory
+2. Follow Fable binding conventions:
+   ```fsharp
+   // src/YourProject.fs
+   module YourProject
+   
+   open Fable.Core
+   open Fable.Core.JsInterop
+   
+   // Import JavaScript module/function
+   [<Import("something", from="some-js-library")>]
+   let something: unit -> unit = jsNative
+   
+   // Define a type that wraps a JS class
+   type MyJsClass =
+       [<Emit("new MyJsClass($0)")>]
+       abstract Create: options: obj -> MyJsClass
+       
+       [<Emit("$0.doSomething($1)")>]
+       abstract DoSomething: parameter: string -> unit
+   ```
```

Comment:
We could also mention https://glutinum.net/ even if they are using the template they are probably aware of it but at least they will have a direct link for it,